src/controllers/initializer.ts

```diff
@@ -24,7 +25,10 @@ export async function create(
     spinner.succeed();
   } else {
     spinner.info('Authenticate to continue');
-    await retrieveQR(waPage);
+    const { data } = await retrieveQR(waPage);
+    if (catchQR) {
+      catchQR(data);
```

Comment:
I would prefer to have an event for this. In case the client disconnects how do I catch the new QR ?